server/routes/applications/applicationDetailsRoutes.ts

```diff
@@ -35,18 +36,34 @@ export default function applicationDetailsRoutes({ auditService }: { auditServic
     asyncMiddleware(async (req: Request, res: Response) => {
       const { applicationData } = req.session
 
+      const applicationType = getApplicationType(req.session.applicationData?.type.apiValue)
+      const fieldToValidate = req.body.swapVosPinCreditDetails
+
       const isSwapVOsToPinCredit =
         applicationData?.type?.apiValue ===
```

Comment:
Just spotted this but this can be swapped to use `applicationType` that gets set on line 39.